src/clients/BinariesVersionClient.ts

```diff
@@ -92,11 +92,11 @@ export class BinariesVersionClient {
     private getOsString(): string {
         switch (process.platform) {
             case `win32`:
-                return `win`;
+                return process.arch && (process.arch === 'arm' || process.arch === 'arm64') ? `win_arm64` : `win`;
             case `darwin`:
-                return `osx`;
+                return process.arch && (process.arch === 'arm' || process.arch === 'arm64') ? `osx_arm64` : `osx`;
```

Comment:
🗞️☕️ Please do verify: document reference for not all arm arch with be osx_arm64 ?

Reply:
Thanks @Tatsinnit I will update this and create another new issue to support other arm arch's like arm, arm32. I initially thought architecture will come back sometimes as arm instead of arm64 which needs testing in the actual arm machine.